src/Sinch/SinchOptions.cs

```diff
@@ -38,6 +39,54 @@ public sealed class SinchOptions
 
         /// <inheritdoc cref="ApiUrlOverrides"/>
         public ApiUrlOverrides ApiUrlOverrides { get; set; }
+
+
+        internal ServicePlanIdOptions ServicePlanIdOptions { get; private set; }
+
+        /// <summary>
+        ///     Use SMS API with `service plan id` and compatible region.
+        ///     `service_plan_id` will be used in place of `project_id`
+        /// </summary>
+        /// <param name="servicePlanId">Your service plan id</param>
+        /// <param name="hostingRegion">Region to use.</param>
+        /// <param name="apiToken"></param>
+        /// <exception cref="ArgumentNullException">throws if service plan id or region is null or an empty string</exception>
+        public void UseServicePlanIdWithSms(string servicePlanId, SmsServicePlanIdHostingRegion hostingRegion,
```

Comment:
The order of parameters seems strange: the documentation says "Default server location is USA" so I would expect the hosting region parameter to be optional.
On the other hand, the apiToken is mandatory and I would expect to see it next to the servicePlanId

Reply:
Makes sense, I'm updating the method signature

Reply:
Also, SmsServicePlainIdHosting regions cannot be made optional due to language limitations